examples/albert/deepspeed_hf.py

```diff
@@ -112,7 +112,7 @@ def loss_fn(outputs, labels):
             lm_loss = lm_loss.contiguous().mean()
             return lm_loss
 
-        model, _ = slapo.build(
+        (model, _), sch = slapo.build(
```

Comment:
Why do we need to return a schedule? Users can still access `sch` even we don't return it

Reply:
i think a different sch object is created in the build?